apps/client/src/common/components/input/popover-picker/PopoverPicker.module.scss

```diff
@@ -14,3 +17,35 @@
   box-shadow: 0 0 0 1px $blue-300;
 }
 
+.icon {
+  border-radius: 99px;
+  border: 0px solid black;
+  box-shadow: 0 0 0 0px;
+  width: 1.75rem;
+  height: 1.75rem;
+}
+
+.icon:hover {
+  box-shadow: 0 0 0 0px;
+}
+
```

Comment:
I tried out some changes, and you were right about not needing the icon class in the first place. Removed that in the changes. Doing another push.

I believe the merge conflicts were sorted previously, in case another shows up, of course we'll fix that too